src/main/java/de/tum/in/www1/artemis/security/annotations/EnforceFile.java

```diff
@@ -0,0 +1,22 @@
+package de.tum.in.www1.artemis.security.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.springframework.security.access.prepost.PreAuthorize;
+
+/**
+ * This annotation is used to enforce an endpoint using a file token authentication.
+ * It should only be used with endpoints starting with {@code /api/file/}
```

Comment:
```suggestion
 * It should only be used with endpoints starting with {@code /api/files/}.
```
Should the convention be `/api/file/` or `/api/files/`? Here it’s singular, the `FileResource` uses the plural variant.

Reply:
My bad